docset/winserver2022-ps/activedirectory/Get-ADComputer.md

```diff
@@ -11,52 +11,71 @@ title: Get-ADComputer
 # Get-ADComputer
 
 ## SYNOPSIS
+
 Gets one or more Active Directory computers.
 
 ## SYNTAX
 
 ### Filter (Default)
+
 ```powershell
-Get-ADComputer [-AuthType <ADAuthType>] [-Credential <PSCredential>] -Filter <String> [-Properties <String[]>]
- [-ResultPageSize <Int32>] [-ResultSetSize <Int32>] [-SearchBase <String>] [-SearchScope <ADSearchScope>]
- [-Server <String>] [<CommonParameters>]
+Get-ADComputer [-AuthType <ADAuthType>] [-Credential <PSCredential>] 
+  -Filter  <String> [-Properties <String[]>] [-ResultPageSize <Int32>]
+  [-ResultSetSize <Int32>] [-SearchBase <String>] [-SearchScope <ADSearchScope>]
+  [-Server <String>] [<CommonParameters>]
 ```
 
 ### Identity
+
 ```powershell
-Get-ADComputer [-AuthType <ADAuthType>] [-Credential <PSCredential>] [-Identity] <ADComputer>
- [-Partition <String>] [-Properties <String[]>] [-Server <String>] [<CommonParameters>]
+Get-ADComputer [-AuthType <ADAuthType>] [-Credential <PSCredential>] 
+  [-Identity] <ADComputer> [-Partition <String>] [-Properties <String[]>]
+  [-Server <String>] [<CommonParameters>]
 ```
 
 ### LdapFilter
+
 ```powershell
 Get-ADComputer [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <String>
- [-Properties <String[]>] [-ResultPageSize <Int32>] [-ResultSetSize <Int32>] [-SearchBase <String>]
- [-SearchScope <ADSearchScope>] [-Server <String>] [<CommonParameters>]
+  [-Properties <String[]>] [-ResultPageSize <Int32>] [-ResultSetSize <Int32>]
+  [-SearchBase <String>] [-SearchScope <ADSearchScope>] [-Server <String>] 
+  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Get-ADComputer** cmdlet gets a computer or performs a search to retrieve multiple computers.
-
-The *Identity* parameter specifies the Active Directory computer to retrieve.
-You can identify a computer by its distinguished name, GUID, security identifier (SID) or Security Accounts Manager (SAM) account name.
-You can also set the parameter to a computer object variable, such as `$<localComputerobject>` or pass a computer object through the pipeline to the *Identity* parameter.
 
-To search for and retrieve more than one computer, use the *Filter* or *LDAPFilter* parameters.
-The *Filter* parameter uses the PowerShell Expression Language to write query strings for Active Directory.
-PowerShell Expression Language syntax provides rich type conversion support for value types received by the *Filter* parameter.
-For more information about the *Filter* parameter syntax, type `Get-Help` [about_ActiveDirectory_Filter](/previous-versions/windows/server/hh531527(v=ws.10)).
-If you have existing Lightweight Directory Access Protocol (LDAP) query strings, you can use the *LDAPFilter* parameter.
+The **Get-ADComputer** cmdlet gets a computer or performs a search to retrieve multiple computers.
```

Comment:
We always want to wrap cmdlet names with backticks so they render as code. Our [style guide][01] has additional formatting guidance you might find useful.


```suggestion
The `Get-ADComputer` cmdlet gets a computer or performs a search to retrieve multiple computers.
```

[01]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide